### PR TITLE
Update & improve tutorial window

### PR DESCRIPTION
**Pull request by Moneyl**
- Fix scroll bar not showing up till window is resized/scrolled.
- Update keybinds and information.
- Add "quick controls" before detailed explanation for quick reference.
- Add section headers with larger font.
- Add paragraph on death & respawning

To make the paragraph formatting look decent I had to make a few lines of code for defining the strings very long. Tried using string linebreaks but they don't seem to work with formatted messages.

Before: (orange A was from me testing formatting, can ignore)
Content Client_Mx9u3byqHV

After:
Content Client_wC9LRkrhAj

### CODE REVIEW

**Comment by RemieRichards** [comment]
If you're listing the controls like this now, why repeat it in the paragraph still?
The list of "Action: Keybind" is much better than the paragraph.

File: Content.Client/UserInterface/GameHud.cs [hunk 1]
```diff
@@ -267,6 +267,8 @@ private void ButtonTutorialOnOnToggled()
             {
                 _tutorialWindow.OpenCentered();
                 _buttonTutorial.Pressed = true;
+                //Fixes scrollbar not showing up until window is resized/scrolled.
+                _tutorialWindow.ForceRunLayoutUpdate();
```
Comment (PJB3005) [inline Content.Client/UserInterface/GameHud.cs]: This really should not be the correct way to fix this issue. The documentation for `ForceRunLayoutUpdate` specifically says that needing to use it most certainly signifies a bug in the UI system, and this is one such place I feel that holds.

Comment (Moneyl) [reply Content.Client/UserInterface/GameHud.cs]: Will see if I can find the issue and make a separate PR with a fix.

**Comment by Moneyl** [comment]
Yeah I agree that most of the paragraph is unnecessary now. I removed most of it except for the sentences on death, respawning, and combat mode since they might not be immediately obvious, and stuck them in a "gameplay" section. 

Content Client_MdgEe4kbeT

Removed the use of `ForceRunLayoutUpdate`. Will try to figure out why the scrollbar isn't showing up until resize and make a separate PR. Will probably be an engine change anyway.

File: Content.Client/UserInterface/TutorialWindow.cs [hunk 2]
```diff
@@ -1,51 +1,90 @@
...
+Toggle entity spawner: [color=#a4885c]F5[/color]
+Toggle tile spawner: [color=#a4885c]F6[/color]
+Toggle sandbox window: [color=#a4885c]B[/color]
 
-Clicking on things ""interacts"" in some object-defined sense with it, with your active hand.
+If you are not on a QWERTY keyboard, the keys mentioned above are bound to the physical location on your keyboard, not what letter they correspond to. For example on AZERTY movement is ZQSD, drop is A, W is activate in hand.
+";
 
-X switches hands. Z uses the item in your hand. Q drops items. T focuses chat. C opens your inventory.
+        private const string GameplayContents = @"Some notes on gameplay. To talk in OOC, prefix your chat message with \[ or /ooc. Death is currently show as a black circle around the player. You can respawn via the respawn button in the sandbox menu. Instead of intents, we have ""combat mode"". Check controls above for it's keybind. You can't attack anybody with it off, so no more hitting yourself with your own crowbar.
```
Comment (PJB3005) [inline Content.Client/UserInterface/TutorialWindow.cs]: Typo on `it's keybind`

Comment (Moneyl) [reply Content.Client/UserInterface/TutorialWindow.cs]: Fixed. Will add a combat mode melee explanation a bit later

**Pull request merged**